hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/common/EventListenersAwareLayout.java

```diff
@@ -26,6 +28,15 @@ default void restoreState() {
     default void onViewEnter() {
     }
 
+    /**
+     * Set components state based on url parameters
+     * 
+     * @param stateParameters
+     *            map of state url parameters
+     */
+    default void handleParameters(final Map<String, String> stateParameters) {
```

Comment:
[Optional] Maybe rename `handleParameters` to `handleStateParameters`?

Reply:
done